File: VHELSING/vhelsing.c
#include <stdio.h>
#include <math.h>
int main() {
    int i;
    double r;
    scanf("%d",&i);
    while(i) {
        scanf("%lf",&r);
        printf("%.4lf\n", 8*(2-sqrt(2))*r*r*r );
        --i;
    }
    return 0;
}

File: VHELSING/vhelsing.go
package main

import "fmt"
import "math"

func main() {
    var i int
	var r float64
	fmt.Scanf("%d",&i)
	for ; i>0; i-- {
		fmt.Scanf("%f", &r)
		fmt.Printf("%.4f\n", 8*(2-math.Sqrt(2))*r*r*r)
	}
}